plone/app/dexterity/behaviors/exclfromnav.py

```diff
@@ -6,7 +6,36 @@
 from z3c.form.interfaces import IAddForm
 from z3c.form.interfaces import IEditForm
 from zope import schema
+from zope.interface import implementer
+from zope.interface import Interface
 from zope.interface import provider
+from zope.schema.interfaces import IContextAwareDefaultFactory
+
+
+class IExcludeFromNavigationDefault(Interface):
+
+    def __call__():
+        """boolean if item is by default excluded from navigation or not.
```

Comment:
PEP 257, follow it now so we don't have to fix it later: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings